coralproject/talk
# Stream Settings

## What does this PR do?
- Adds a Settings icon to configure the Stream

## How do I test this PR?

- Go to Moderate
- Moderate a Stream
- You should see a `Settings Icon` a cog
- It should go to the Asset Url
